File: skysail.server.app.ref.singleentity/test/io/skysail/server/app/ref/singleentity/RunCucumberTests.java
package io.skysail.server.app.ref.singleentity;

import org.junit.runner.RunWith;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;

@RunWith(Cucumber.class)
@CucumberOptions(
        //features
        //tags = {"@localtests"},
        tags = {"@JustMe"},
        //tags = {"@singleEntityRefApplication"},
        glue   = { "io.skysail.server.app.ref.singleentity" },
        plugin = { "pretty", "json:generated/cucumber.json", "html:generated/cucumber" }
)
public class RunCucumberTests { // NOSONAR

}

File: skysail.app.notes/test/io/skysail/app/notes/test/RunCucumberTests.scala
package io.skysail.app.notes.test


import org.junit.runner.RunWith;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;

@RunWith(classOf[Cucumber])
@CucumberOptions(
      tags = Array("@notesApplication"),
      glue   = Array("io.skysail.app.notes.test"),
      plugin = Array("pretty", "json:generated/cucumber.json", "html:generated/cucumber")
)
class RunCucumberTests { }
